### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 8th of August 2023 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#1111] [Flatcar Office Hours]
- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://youtube.com/live/omppbFNxSDU
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- @tormath1 FrOSCon recap and demo of systemd sysext kubernetes image (with CAPO or locally) using the latest sysext-bakery release to use plain Flatcar images in Cluster API

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

### Open Floor
- @miao0miao  Bug smashing day in September 

### Administrative tasks before meeting starts
- [ ] Create a comment with the list of closed/opened issues/PRs (hint: also sort by time updated)
### Administrative tasks after meeting concluded
- [ ] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [ ] Close this discussion as "Resolved".

**Comment by tormath1** [comment]
I can briefly mention the FrOSCon event and a quick demo of systemd sysext kubernetes image (with CAPO or locally) using the latest sysext-bakery release to use plain Flatcar images in Cluster API (cc @mdbooth, if you are around to see what we discussed in the last CAPO office hours)
